src/pages/playground/Playground.tsx

```diff
@@ -104,6 +107,36 @@ export type StateProps = {
 
 const keyMap = { goGreen: 'h u l k' };
 
+function handleHash(hash: string, props: PlaygroundProps) {
```

Comment:
Nice this is actually cleaner than the previous function `parsePlayground`, which honestly did not need the entire list of helper parsing functions.